include/networkit/centrality/KatzCentrality.hpp

```diff
@@ -12,6 +12,8 @@
 
 namespace NetworKit {
 
+enum KatzVariant { InEdges = 0, OutEdges = 1 };
```

Comment:
Do not call this KatzVariant (The same could be done in the case of PageRank, too) but something like EdgeDirection.

Reply:
Agree, I changed the name